openapi_builder/__meta__.py

```diff
@@ -2,7 +2,7 @@
 path = name.lower().replace("-", "_").replace(" ", "_")
 # The version number should follow https://python.org/dev/peps/pep-0440 and
 # https://semver.org
-version = "0.2.10"
+version = "0.2.11"
```

Comment:
I would create 0.3.0 since there are internal changes affecting public output. But since we're the only users it doesn't really matter.